### DISCUSSION TITLE
Add contact sensors for Salt and Rise aid (Dishwasher)

### DISCUSSION THREAD

**Discussion by Bingo269** [thoukydides/homebridge-homeconnect#150] [Ideas]
Instead of having buttons in HomeKit, which is great for automations, it would be very useful to have two contact sensors additionally which are exposing if the rinse aid or salt reservoir has to be refilled.

**Comment by thoukydides** [comment]
Sorry, but no.

Home Connect only reports conditions like low salt and rinse aid as events, rather than persistent status. It does not allow the current status of these supplies to be read. Hence, it only makes sense mapping these events to stateless buttons.

Anyway, I don't really see the point. The only reasons I see for integrating these appliances with HomeKit are:
* To allow control from Siri, shortcuts, or automations.
* To trigger automations.

The actual interface within the Home app is rubbish for appliances like these for which there isn't a suitable model. It is always better to use the official Home Connect app.

**Reply by markhw001** [reply]
Hi, I have a Siemens washer WG56B2A1GB/11. This machine has i-Dos 1 (container 1) and i-Dos 2 (container 2) there are "nearly empty" warnings available. Could these be added to the Homeassistant Home Connect integration.

**Reply by thoukydides** [reply]
This project is a Homebridge plugin. It does not have any connection with HomeAssistant.
